docs/user/man_pages/bodhi.rst

```diff
@@ -54,6 +54,10 @@ Commands
 There are two commands, ``overrides`` and ``updates``. They are described in more detail in their
 own sections below.
 
+``bodhi composes <subcommand> [options] [args]``
```

Comment:
I don't see the composes command anywhere in this man page. There is section for `overrides`, `releases` and `updates`. But no section for `composes`.